awl/mslider.cpp

```diff
@@ -229,20 +229,27 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
       //    draw tick marks
       //---------------------------------------------------
 
-  	QFont f(p.font());
-   	f.setPointSize(6);
+   QFont f(p.font());
+      f.setPointSize(6);
    	p.setFont(f);
       p.setPen(QPen(Qt::darkGray, 2));
    	QFontMetrics fm(f);
-      int xt = 20 - fm.width("00") - 5;
+      int xt = 20 - fm.width("-00") - 5;
 
       QString s;
-   	for (int i = 10; i < 70; i += 10) {
-      	h  = y1 + lrint(i * mh / range);
-         	s.setNum(i - 10);
-  		p.drawText(xt,  h - 3, s);
-		p.drawLine(15, h, 20, h);
-         	}
+      for (int i = 0; i <= (maxValue() - minValue()); i += 10) {
+            h  = y1 + lrint(i * mh / range);
+            s.setNum(i);
+            if (i == 0) {
+                  p.drawText(QRect(xt - 9, h - 3, 20, 9), Qt::AlignRight, s + QString::fromStdString("dB"));
```

Comment:
like I said, well have to get rid of the "s +" part here so can declare victory.